config/rbac/role.yaml

```diff
@@ -8,12 +8,13 @@ rules:
 - apiGroups:
   - apps
   resources:
-  - deployments
+  - statefulsets
```

Comment:
so correct me if I'm wrong but keystone does not store any state in the container or in persistent volumes.

all its state is stored in the db correct? has that changed with fernet tokens? we should avoid stateful sets for no stateful services so I want to understand if you are making this a statefulset because you need each instance of keystone to use a sepert set of pvc or not.

Reply:
its only because I like having the pods named keystone-0, -1 ,-2 instead of the deployment naming keystone-xyzpb. thats the only reason. That would make it easier to see the number of running instances. we can switch it back to be a deployment.

Reply:
moved back to use a deployment